gammapy/data/tests/test_event_list.py

```diff
@@ -19,6 +19,7 @@ def test_EventList():
     assert event_list.radec[0].to_string() == '82.7068 19.8186'
     assert event_list.galactic[0].to_string() == '185.956 -7.69277'
     assert event_list.altaz[0].to_string() == '46.2059 31.2001'
+    assert_allclose(event_list.offset[0].value, 1.904497742652893)
```

Comment:
ok Id’ont understand why the test is working for me and not for you!
Le 1 févr. 2016 à 17:52, Christoph Deil <notifications@github.com> a écrit :

> assert_allclose(event_list.offset[0].value, 1.904497742652893, rtol=1e-5)